tests/Doctrine/Tests/Common/Cache/CacheTest.php

```diff
@@ -96,6 +96,31 @@ public function testFetchMultiWithEmptyKeysArray()
         );
     }
 
+    public function testFetchMultiWithFalsey()
+    {
+        $cache = $this->_getCacheDriver();
+
+        $cache->deleteAll();
+
+        $values = array(
```

Comment:
This should have used `provideDataToCache` instead which could be reused and also covers more.